source/assets/stylesheets/sections/_code_of_conduct.scss

```diff
@@ -0,0 +1,7 @@
+body.code-of-conduct {
+  // TODO: Use these sizes for the whole website
+  @media (max-width: $screen-xs-max) { font-size: 16px; }
```

Comment:
These sizes should become variables as they seemed to be used again [here](https://github.com/brug-be/rubybelgium/pull/82/files#diff-41aca4d97598d89461ad2e4d26d4fd39R13) for example